corehq/util/urlsanitize/test/test_urlsanitize.py

```diff
@@ -18,10 +20,10 @@ def setup_module():
 
     GOOGLE_IP = ipaddress.ip_address(socket.gethostbyname('google.com'))
     SUITE = [
-        ('https://google.com/', (RETURN, GOOGLE_IP)),
-        ('http://google.com/', (RETURN, GOOGLE_IP)),
-        ('http://google.com', (RETURN, GOOGLE_IP)),
-        ('http://foo.example.com/', (RAISE, CannotResolveHost())),
+        ('https://google.com/', (RETURN, 'ignored')),
+        ('http://google.com/', (RETURN, 'ignored')),
+        ('http://google.com', (RETURN, 'ignored')),
```

Comment:
Updated. I opted against using `None` because it wasn't really an important part of what this test was trying to verify. If this function was amended to return a value for convenience, this test shouldn't care -- it just wants to verify that valid addresses do not raise exceptions.